lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -1089,7 +1090,7 @@ poolRegistrationsFromBlock (Block _hdr fragments) = do
     let margin = fromRight maxBound $ mkPercentage $ toRational $ taxRatio taxes
     let cost = Quantity (taxFixed taxes)
     let dummyPledge = Quantity 0
-    pure $ W.PoolRegistrationCertificate poolId owners margin cost dummyPledge Nothing
+    pure $ W.PoolRegistrationCertificate poolId owners margin cost dummyPledge Nothing NoPoolFlag
```

Comment:
**Request:** move constructor arguments onto a separate line. (See [line limit](https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters).)

**Suggestion:** give a name to the metadata argument to make it more obvious what this argument is.

```suggestion
    let metadata = Nothing
    pure $ W.PoolRegistrationCertificate
        poolId owners margin cost dummyPledge metadata NoPoolFlag
```